<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Airplanner</class>
 <widget class="QMainWindow" name="Airplanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>677</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Agency FB</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Airplanner</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(135,206,250	);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SubmitButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>630</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(135,206,250);
border-top: 1px solid rgb(10, 18, 27);
font: 10pt &quot;Lucida Sans Unicode&quot;;
color: rgb(250, 250, 250);</string>
    </property>
    <property name="text">
     <string>增加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RemoveButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>630</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(135,206,250);
border-top: 1px solid rgb(10, 18, 27);
font: 10pt &quot;Lucida Sans Unicode&quot;;
color: rgb(250, 250, 250);</string>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>630</y>
      <width>361</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(135,206,250);
border-top: 1px solid rgb(10, 18, 27);
padding-top: 8px;
font: 10pt &quot;Lucida Sans Unicode&quot;;
color: rgb(250, 250, 250);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Sans Unicode'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>请输入新任务</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(135,206,250);
border: 0;
border-bottom: 1px solid rgb(10, 18, 27);
font: 10pt &quot;Lucida Sans Unicode&quot;;
color: rgb(250, 250, 250);</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="MenuBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(135,206,250);
border: 0;
border-bottom: 1px solid rgb(10, 18, 27);
padding-left: 5px;
font: 12pt &quot;Impact&quot;;
color: rgb(135,206,250);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>60</y>
      <width>451</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
padding-left: 5px;
font: 10pt &quot;Lucida Sans Unicode&quot;;
color: rgb(250, 250, 250);</string>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
   </widget>
   <widget class="QLabel" name="MenuBar_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(135,206,250);
border: 0;
border-bottom: 1px solid rgb(10, 18, 27);
padding-left: 5px;
font: 15pt &quot;Impact&quot;;
color: rgb(250, 250, 250);</string>
    </property>
    <property name="text">
     <string>个人todo</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
